jobs/JAQM_MAKE_ICS

```diff
@@ -32,72 +22,18 @@ export CDATE=${PDY}${cyc}
 
 . $USHaqm/job_preamble.sh
 
-export EXECaqm=${EXECaqm:-${HOMEaqm}/exec}
-export FIXaqm=${FIXaqm:-${HOMEaqm}/fix}
-
-export COMINgfs="${COMINgfs:-$(compath.py gfs/${gfs_ver})}"
-export COMIN="${COMIN:-$(compath.py ${NET}/${aqm_ver}/${RUN}.${PDY})}"
-export COMINm1="${COMINm1:-$(compath.py ${NET}/${aqm_ver}/${RUN}.${PDYm1})}"
-export COMOUT="${COMOUT:-$(compath.py -o ${NET}/${aqm_ver}/${RUN}.${PDY}/${cyc})}"
 
-#-----------------------------------------------------------------------
-# Get the full path to the file in which this script/function is located 
-# (scrfunc_fp), the name of that file (scrfunc_fn), and the directory in
-# which the file is located (scrfunc_dir).
-#-----------------------------------------------------------------------
-#
-scrfunc_fp=$( $READLINK -f "${BASH_SOURCE[0]}" )
-scrfunc_fn=$( basename "${scrfunc_fp}" )
-scrfunc_dir=$( dirname "${scrfunc_fp}" )
+cycle_iso=${PDY:04}-${PDY:4:2}-${PDY:6:2}T${cyc}
+uw chgres_cube provisioned_rundir \
+  --cycle $cycle_iso \
+  --config ${USHaqm}/config.yaml \
+  --key-path make_ics
 
-#env
-#-----------------------------------------------------------------------
-# Print message indicating entry into script.
-#-----------------------------------------------------------------------
-#
-print_info_msg "
-========================================================================
-Entering script:  \"${scrfunc_fn}\"
-In directory:     \"${scrfunc_dir}\"
-
-This is the J-job script for the task that generates initial condition 
-(IC), surface, and zeroth-hour lateral boundary condition (LBC0) files 
-for the FV3 (in NetCDF format).
-========================================================================"
-#
-#-----------------------------------------------------------------------
-# Set the name of and create the directory in which the output from this
-# script will be placed (if that directory doesn't already exist).
-#-----------------------------------------------------------------------
-#
-export INPUT_DATA="${INPUT_DATA:-${COMIN}/${cyc}}"
-mkdir -p ${INPUT_DATA}
-#
-#-----------------------------------------------------------------------
-# Set the run directory
-#-----------------------------------------------------------------------
-#
-DATA="${DATA:-${COMIN}${SLASH_ENSMEM_SUBDIR}/tmp_MAKE_ICS}"
-#
-#-----------------------------------------------------------------------
-# Set environment only when RUN_TASK_GET_EXTRN_ICS is false.
-#-----------------------------------------------------------------------
-#
-if [ "${RUN_TASK_GET_EXTRN_ICS}" = "FALSE" ]; then
```

Comment:
Maybe this is still used in some cases, but it seems like it might be more straightforward just to have this script always handle this. 

Again, this is removed for demo purposes only.